src/currency.js

```diff
@@ -1,14 +1,14 @@
 define([
 	"./core",
+	"./common/runtime-bind",
 	"./common/validate/cldr",
 	"./common/validate/default-locale",
 	"./common/validate/parameter-presence",
 	"./common/validate/parameter-type/currency",
 	"./common/validate/parameter-type/number",
 	"./common/validate/parameter-type/plain-object",
-	"./common/validate/plural-module-presence",
 	"./currency/code-properties",
-	"./currency/name-format",
+	"./currency/formatter/fn",
```

Comment:
EDIT: This is supposed to be the returned function by the generator. Take for example `numberFormatter`, it returns a formatter, that's what this function is about.

I am going to name it `./currency/formatter-fn` and make the same for all other analogous. Feel free to suggest other alternatives.